pkg/core/pipeline/autodiscovery/config/main.go

```diff
@@ -11,8 +11,8 @@ type Config struct {
 	Crawlers map[string]interface{} `yaml:",omitempty"`
 	// ScmID specifies a scmid configuration to use to generate the manifest
 	ScmId string `yaml:",omitempty"`
-	// PullrequestID specifies a pullrequest configuration to use to generate the manifest
-	PullrequestId string `yaml:",omitempty"`
+	// ActionId specifies an action configuration to use to generate the manifest
+	ActionId string `yaml:",omitempty"`
```

Comment:
Ping @olblak : is it worth it to add a depreciation message? I did not assuming autodiscovery is still experimental so it can change, but if you feel like we should, no problem to add a message for this one (since there is an E2E test with the `pullrequestid` attribute)

Reply:
Well if it doesn't take you much time, I would prefer having a deprecation message